src/views/workflow-summary-tab/workflow-summary-tab-json-view/workflow-summary-tab-json-view.tsx

```diff
@@ -47,10 +53,22 @@ export default function WorkflowSummaryTabJsonView({
           overrides={overrides.copyButton}
         />
       </div>
-      {activeTab === 'result' && isWorkflowRunning ? (
-        <PrettyJsonSkeleton width="200px" />
-      ) : (
-        <PrettyJson json={jsonMap[activeTab]} />
+      {activeTab === 'input' && <PrettyJson json={jsonMap[activeTab]} />}
+      {activeTab === 'result' && isArchived && (
+        <div className={cls.archivedResult}>
+          Workflow is archived, result is only available in{' '}
+          <Link
+            href={`/domains/${domain}/${cluster}/workflows/${workflowId}/${runId}/history`}
+          >
+            history
+          </Link>
+        </div>
+      )}
+      {activeTab === 'result' && !isArchived && (
+        <>
+          {isWorkflowRunning && <PrettyJsonSkeleton width="200px" />}
```

Comment:
Maybe for later, but we can replace this placeholder with a more useful "workflow is still running" message